ui/src/shared/utils/analytics.ts

```diff
@@ -0,0 +1,25 @@
+export const fireOrgIdReady = function fireOrgIdReady(
+  organizationIds: string[]
+) {
+  window.dataLayer = window.dataLayer || []
+  window.dataLayer.push({
+    event: 'cloudAppOrgIdReady',
+    identity: {
+      organizationIds,
+    },
+  })
+}
+
+export const fireUserDataReady = function fireUserDataReady(
```

Comment:
Really interesting setting a const to a function this way. I am curious as to why the parameter is being passed a `user.name` but is expecting an `email`. Is the parameter the source of truth here (in that the `user.name` is an email)?

Reply:
>Really interesting setting a const to a function this way. 

@Asalem1 it's yavascript learnin' time with uncle bucko

```ts
const foo = function() {
  console.log('anonymous')
}
```
creates an anonymous function and assigns it to the variable `foo`

```ts
const bar = function fineYoungCannibals() {
   console.log('fineYoungCannibals')
}
```
creates a function named `fineYoungCannibals` and assigns it to the variable `bar`. On stack traces, this name will show up, rather than nothing. Check [the first example](https://hackernoon.com/three-reasons-i-avoid-anonymous-js-functions-like-the-plague-7f985c27a006) in this article to see what i mean.

I named the variable `bar` and the function `fineYoungCannibals` to show that the naming of a function and the assigning of a function to a variable are separate things that in the instance above, happen to share the same name.

---

to answer your question re: `user.name` and `email.`

`user.name` seems to represent username, which is what is entered into the form. it doesn't need to be an email (e.g. locally, my login is `bucky` | `password` [plz don't hax0r me]), but for our intents, it -should- be email. (this is confusingly named, not sure about the context behind it).